Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.h

```diff
@@ -30,13 +32,24 @@ namespace AzToolsFramework::Prefab
 
         // PrefabFocusInterface override ...
         PrefabFocusOperationResult FocusOnOwningPrefab(AZ::EntityId entityId) override;
+        PrefabFocusOperationResult FocusOnPathIndex(int index) override;
         TemplateId GetFocusedPrefabTemplateId() override;
         InstanceOptionalReference GetFocusedPrefabInstance() override;
         bool IsOwningPrefabBeingFocused(AZ::EntityId entityId) override;
+        const AZ::IO::Path& GetPrefabFocusPath() override;
+        const int GetPrefabFocusPathLength() override;
+
+        // EditorEntityContextNotificationBus...
```

Comment:
nit: Prefer `EditorEntityContextNotificationBus overrides ...` style (could fix other usages in this file too)